entries/m/msufcu.org.json

```
@@ -0,0 +1,14 @@
+{
+    "MSU Federal Credit Union": {
+        "domain": "msufcu.org",
+        "tfa": [
+            "sms",
+            "call",
+            "email"
+        ],
+        "documentation": "https://www.msufcu.org/securitycenter",
+        "keywords": [
+            "banking"
+        ]
```

As only US residents can be customers a regions element is also needed.
```suggestion
        ],
        "regions": [
          "us"
        ]
```